jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -280,6 +280,82 @@ static const uint8_t ecma_uint4_clz[] = { 4, 3, 2, 2, 1, 1, 1, 1, 0, 0, 0, 0, 0,
  */
 #define EPSILON 0.0000001
 
+/**
+ * ECMA-defined conversion of string to Number.
+ *
+ * String to number for different bases (2,8,16)
+ * See also:
+ *          ECMA-262 v5, 9.3.1
```

Comment:
Ocal and binary conversions are only present since the v6 standard so please mention the related standard point.